VideoWeb/VideoWeb/ClientApp/src/app/shared/directives/tooltip.directive.ts

```diff
@@ -1,11 +1,13 @@
 import { Directive, ElementRef, EventEmitter, HostListener, Input, OnDestroy, Output, Renderer2 } from '@angular/core';
+import { DeviceTypeService } from 'src/app/services/device-type.service';
 
 @Directive({
     selector: '[appTooltip]'
 })
 export class TooltipDirective implements OnDestroy {
     _text: string;
     _colour = 'blue';
+    _canShowInMobile = false;
```

Comment:
I am not sure there is a isTouch in any library that we use. But we should be able to check using https://developer.mozilla.org/en-US/docs/Web/API/Navigator/maxTouchPoints

However, this doesn't guarantee that the user isn't actually using another input device where we want to show a tool tip.

Reply:
Yeah. I'm thinking we shouldn't have it based on device at all. 

I know Angular Material's tooltip handled touch and hover out of the box so maybe we could steal something from that.

Maybe we could look at adding additional events as well as mouseenter and mouseleave. Like pointerdown on the element to display and pointerdown outside the element to hide
https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events

Not as part of this ticket but it's worth noting that we also need to consider keyboard only users. Can we provide all this information to them as well?

Reply:
Ah I didn't realise the issue was that it shows up on touch but stays until the user touches elsewhere. So essentially what I was saying already happens here and this change is more about the timeout.

Reply:
Discussed this with Zijian yesterday. While it's not a perfect solution as this isn't a mobile vs desktop issue but rather an input device issue. Using different events is not straight forward enough to merit implementing here and would be more valid as tech debt. We also have other issues with tooltips here and I think this solution is ok in most cases. We should probably look at importing a mature tooltip solution from npm that handles things like touch and keyboard only rather than implementing it ourselves.